src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -14348,11 +14371,15 @@ void Player_Action_8084F88C(Player* this, PlayState* play) {
     }
 }
 
-void Player_Action_8084F9A0(Player* this, PlayState* play) {
+/**
+ * Automatically open a door (no need for the A button).
+ * Note: If no door is in useable range, a softlock will occur.
+ */
+void Player_Action_OpenDoor(Player* this, PlayState* play) {
```

Comment:
I would have expected `Player_Action_80845EF8` to take the name `Player_Action_OpenDoor`, is there a better way to distinguish between these two action funcs?

Reply:
Thats more like using a door, from open -> walk through -> close (if knobbed) -> go to idle
So I would expect that one to be named something like `Player_Action_UseKnobDoor`

Sliding doors dont even have their own action, they just use `Player_Action_80845CA4` which is the auto-walk action (shared with, for example, the walking start modes)

Reply:
`Player_Action_80845EF8` seems specific to opening doors with a handle. It does not run when sliding doors are opened (those switch to `Player_Action_80845CA4`).

Reply:
Yeah, mentioned above

Reply:
Ah, that's good to know, thanks. I think I like `Player_Action_AutoOpenDoor` more than `Player_Action_OpenDoor` but it's not a big deal.

Another thought is to tie this and other action names to the `Start` word. So something like:
`Player_Action_StartFromWarpSong`
`Player_Action_StartFromBlueWarp`
`Player_Action_StartFromFaroresWind`
`Player_Action_StartFromTimeTravel` (maybe with Arrival appended on the end of those last four)
`Player_Action_StartFromOpenDoor`
`Player_Action_StartFromGrottoExit`
To show that these are start mode actions. Thoughts?

Reply:
I think I avoided something like that cause, to me, the actions arent *necessarily* strictly for "starting".
Like some of them are by nature, like arriving from a warp song. But the door one for example, can easily be used in some other context that isnt a start mode. 
All the start-modey initial stuff would happen in the `_StartMode_` setup function.

Reply:
As far as this function name goes, we can do `Player_Action_TryOpeningDoor`
Since he does try over and over again, and will fail (softlock) if there is no door

Reply:
I went with this for now